packages/hardhat/hardhat.config.ts

```diff
@@ -121,6 +121,14 @@ const config: HardhatUserConfig = {
       url: "https://sepolia.publicgoods.network",
       accounts: [deployerPrivateKey],
     },
+    celo: {
+      url: "https://forno.celo.org",
+      accounts: [deployerPrivateKey],
+    },
+    celoAlfajores: {
+      url: "https://forno.alfajores.celo-testnet.org/",
```

Comment:
I think this RPC url doesn't work? 

tried doing `yarn deploy --network celoAlfajores`



Should it be https://alfajores-forno.celo-testnet.org ?